.github/workflows/ci.yml

```
@@ -36,14 +36,17 @@ jobs:
 
       - name: Install Poetry
         run: |
-          pip install wheel
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+          pip install --user poetry
```

Thoughts about using `pipx` instead? I guess that would be better.
```suggestion
          pipx install poetry
```

---

I've spent an hour getting this stuff to work with all platforms. Don't touch it!

---

But honestly, I don't mind.

---

> I've spent an hour getting this stuff to work with all platforms. Don't touch it!

You could have asked me beforehand :D